typescript/sdk/src/token/checker.ts

```diff
@@ -54,7 +54,11 @@ export class HypERC20Checker extends ProxiedRouterChecker<
 
       // XERC20s are Ownable
       const expectedConfig = this.configMap[chain];
-      if (expectedConfig.type === TokenType.XERC20Lockbox) {
+
+      if (
+        expectedConfig.type === TokenType.XERC20Lockbox &&
+        expectedConfig.ownerOverrides?.collateralToken
+      ) {
```

Comment:
`expectedConfig.ownerOverrides?.collateralToken`
can you pull this condition out into the parent conditional?
these if/else branches should just be on token type

Reply:
fixed